CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.0.1
```

Comment:
I wasn't sure if we should release this as a patch or a minor. 🤷

Reply:
I think it should be a minor since it adds a flag 🤷  which I would consider an addition to an api, but I'm not well versed in versioning of cli tools

Reply:
Just fyi, I updated MARV to cut a minor release (`2.1.0`).